styles/main.css

```diff
@@ -14,17 +14,18 @@ header {
     margin: 0 -2em 0;
 }
 header nav {
-    float: right;
     box-sizing: content-box;
     margin: -4px 2em 0;
     border-bottom: 1px dotted rgba(0, 0, 0, 0.3);
+    overflow: auto;

```

Comment:
This is the property that makes `nav` take on the height of its floated contents